data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Recent anti-pirate actions by the Free Worlds and Republic Navy have weakened the large anarchist organizations that had deals with the Hai. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations, haven't worked with small pirate groups.`
+			`	They can't track small pirate groups, but they clearly have an intelligence network. Is it good for anything?`
+				goto reading
+				to display
+					not "Hai Reveal [A11] Sound the Alarm: human book"
+			`	It seems that, although the Hai have no trouble with large, established, hierarchies, they have trouble tracking small, independent, anarchist, groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: human book"
+
+			label unfettered
+			action
+				log `A Hai intelligence report puts the cost to Hai of the Unfettered raids at 3.5 billion credits a day, while the cost is expected to double in "full Hai unification." Unification costs include 5 billion credits a day of "military aid."`
+				log `A Hai intelligence report on "Unfettered military activities" included several fully-redacted sections, including "Multi-Decadal Timeline Expected before Unfettered Progress," and "Theories on the Sentient Evolution of Nocturnal Raptors."`
+				set "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report begins with an economic analysis of the cost of raids at Wah Ki, which it places at a staggering 3.5 billion credits per day. This is 1.5 billion impact on trade; 1.5 billion for military fleet repairs, replacements, ammunition, and infrastructure to support it; and an estimated 500 million in humanitarian aid to the Unfettered, to prevent the problem from worsening. It compares that to "full Hai unification" for which the cost is 7 billion credits per day: 3 billion in humanitarian aid to the Unfettered, 5 billion in military aid, but 1 billion per day recouped through improved trade.`
+			`	The remaining 80% of the report is redacted in its entirety, save for a few section headings:`
+			`	-	Multi-Decadal Timeline Expected before Unfettered Progress`
+			`	-	Choot'k: Insane Risk-Taker or Brilliant Strategist?`
+			`	-	Reports from Hai Agents on Unfettered Raiding Vessels`
+			`	-	Unsustainable Rate of Losses Spying on Unfettered Leadership`
```

Comment:
```suggestion
			`	-	Lack Of Success Spying on Unfettered Military Leadership`
```
I feel like it wouldnt be that many losses, more like the fact you cant get any idea of whats going on deep

Lemme explain: the first unfet mission shows Chiefs (Uhai Elders) gathered and answering the populace on many issues
That is smth anyone can get on the unfettered.

However, military stuff would be closely garded. I dont think the agents would be dumb enough to enter the throne rooms of warlords to get intel (at that point they would die yes) but instead would be smarter than that and satisfied with (a bit) of second hand intel?

If you really want to press the matter then it makes sense they would take risks and die but not otherwise
Mb adding recent could work, at the very least Military Leadership is needed

Reply:
The success is limited to sneaking agents onto raiding vessels.

Reply:
I applied your change, but I'm leaving this open for further discussion.

Reply:
You dont get on a raiding vessel without deserving it though
But I can accept the agents being undercover for q long time

this would create frictions though, since the unfetteted would likely sometimes learn about it, which is the opposite of what they want?

Reply:
The Unfettered expect the Fettered to spy on them, and vice-versa. It is how cold wars work. I'm sure they scold each other whenever it happens and go back to the status quo.

Reply:
Alr well its ok since the text was changed - they shouldnt outright tell you they're failing because their agents are dying anyway

Reply:
Yes, you are correct, they should not say that. However, I do want them to warn you that they don't really know what the Unfettered are planning. That's why they don't redact the "Lack Of Success Spying on Unfettered Military Leadership" section heading.